src/authentication/configuration.ts

```diff
@@ -0,0 +1,40 @@
+import * as vscode from 'vscode';
+
+export interface IAppConfiguration {
+    clientId: string | undefined;
```

Comment:
Not a big deal but our existing code is using tabs ;)

Reply:
Darnit, I noticed that and configured prettier to use tabs but looks like spaces still snuck in. Fixicating...